build/Compile.targets

```diff
@@ -124,17 +121,6 @@
             DestinationFiles="@(NETCore10Assemblies)" />
 
       <ItemGroup>
-        <MSBuildTargetsToCopy Include="$(MSBuildTargetsDirectory)/**/*.targets" />
-        <MSBuildTargetsToCopy Include="$(MSBuildTargetsDirectory)/**/*.Targets" />
```

Comment:
Note, this second capitalization was removed. I checked if we have any targets that use it and didn't find any. I removed it because Windows produced duplicate items. This was fine with `copy` because we just copied twice. With `Move`, the duplicate causes a build failure since the file was already moved. I moved to `move` because I'm reducing the amount of unnecessary files in our build output.